### ISSUE TITLE
What Issues and Functionality your missing ?

### ISSUE THREAD

**Issue by geekaryan** [Akshat111111/india-flix#2]
Hy Akshat. I have seen your Netlix clone and it's good and impressive. Can you specify an issues your facing or Can I start helping with you with the Frontend part. I can Implement backend in this application, which make it run smoothly and full stack project. I hope this is benifical and helpful for both of us .
Thank you

**Comment by Akshat111111** [comment]
Yeah brother it will be definitely benificial for both of us.Frontend part is fine,If you will work for the backend part,it would be really very helpful.Thankyou so much for your support ✨✨

**Comment by geekaryan** [comment]
Sure. I will make an API for your application using NodeJs. I will intialize a pull request and make a branch regarding backend. I mean time can you please specify what fields you need for backend.

**Comment by Akshat111111** [comment]
I only want a simple backend which can store the info of user in database.you can use firebase or MySQL for it

**Comment by geekaryan** [comment]
Sure. I will start working on it.
